Sources/FluentUI_common/Core/Theme/Tokens/DynamicColor.swift

```diff
@@ -6,30 +6,31 @@
 import SwiftUI
 
 /// A container that stores a dynamic set of `Color` values.
-struct DynamicColor: Hashable {
+@objc(MSFDynamicColor)
+public final class DynamicColor: NSObject {
```

Comment:
That a bunch of stuff (well actually not that much, but not zero) now has to be exposed publicly when it wasn't before makes me second guess myself. There's no way to share code selectively across platforms with use of path includes/excludes in Package.swift/MicrosoftFluentUI.podspec?

Reply:
Basically, is it actually gross to do this without introducing a new module?

Reply:
I've been beating my head against this for months now, and it's really just a limitation of SPM. If we went back to Cocoapods and Xcode projects alone, this would be very easy to support!

The problem is that Swift Packages only allow gating per-platform on a target (read: module) level. There is no way to conditionally include/exclude specific paths or files as part of a build.

The only options I saw that would work for our needs:
* The one you see here: two modules, more public than we hoped, etc.
* A single module/target, but every platform-specific file needs an `#if` gate around itself, *a la* `UIColor+Extensions.swift` from your earlier comment.
* Drop Swift Package Manager support 🧐

Reply:
Got a solution in place! Cocoapods will continue to work as it has, while SPM will treat `common` as a separate module.